sky/provision/kubernetes/utils.py

```diff
@@ -866,6 +866,15 @@ def check_credentials(context: Optional[str],
 
     _, exec_msg = is_kubeconfig_exec_auth(context)
 
+    # Check whether pod_config is valid
+    pod_config = skypilot_config.get_nested(('kubernetes', 'pod_config'),
+                                            default_value={},
+                                            override_configs={})
+    if pod_config:
+        _, pod_msg = _check_pod_config(context, pod_config)
+        if pod_msg:
+            return False, pod_msg
```

Comment:
Can we have this pod_config check run at every sky launch, and not just at `sky check`?

Reply:
sure, i will change this at every sky launch